docs/features/sinks/azure-application-insights.md

```diff
@@ -57,4 +57,29 @@ if(string.IsNullOrEmpty(key) == false)
 ILogger logger = loggerConfig.CreateLogger();
 ```
 
+### Q: Where can I initialize the logger in an ASP.NET Core application or other hosted service?
+
+Simply use the `UseSerilog` extension method on `IHostBuilder` which accepts an `Action<HostBuilderContext, IServiceProvider, LoggerConfiguration>`.  This Action gives you access to the configuration and the configured services:
+
+```csharp
+var host = Host.CreateDefaultBuilder()
```

Comment:
Should we add the required using statement for our sink?

Reply:
That might be helpful.  Added it.